gmaildump/messagestore.py

```diff
@@ -5,7 +5,7 @@
 import sqlite3
 from deeputil import Dummy
 from diskdict import DiskDict
-from pymongo import MongoClient, ASCENDING, DESCENDING
+from pymongo import MongoClient
```

Comment:
built in modules should be imported first, there should be line spacing between built in and third party.